pkg/util/profiling/profiling.go

```diff
@@ -25,6 +26,8 @@ const (
 	ProfileCoreService = "datadog-agent"
 	// ProfilingLocalURLTemplate is the constant used to compute the URL of the local trace agent
 	ProfilingLocalURLTemplate = "http://%v/profiling/v1/input"
+	// DefaultProfilingPeriod defines the default profiling period
+	DefaultProfilingPeriod = 5 * time.Minute
```

Comment:
I'm wondering if the default profiling period should remain the same, and if instead we should override it to 5 minutes in our deployments configuration.

Reply:
I'm worried we'll forget about it and end up deploying without the manual configuration change once we deploy this more broadly. I updated the configuration of the Process Agent so the period should now be configurable for everyone. Worst case scenario, we can use the configuration parameter to set the clusters on which we want a better granularity ?

Reply:
I agree with @truthbk on this: the default profiling period should remain the same as the one defined in the Go profiler. In particular, the default is 15 minutes today not 5. Setting a custom profiling period should be opt-in only. cc @felixge

Reply:
@flodav The `DefaultPeriod` in [dd-trace-go/profiler](https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/profiler#pkg-constants) is 1 minute, but most dd-go apps seem to use 5 minutes. A higher period is probably a bit friendlier on our backend, especially for aggregation.

Anyway, I have no strong opinions on managing agent configurations. But a 5 minutes default seems pretty reasonable. But in this case it might be worse bumping the DefaultDuration for CPU profiling from 15s to something higher as well (75s?).

Reply:
I mixed up `DefaultPeriod` and `DefaultDuration` 🤦  and also the fact that we are not in the dd-go repository (I thought you were setting the default value  to 5 minutes for all Go applications). My bad.

LGTM